Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -915,7 +917,27 @@ void ACesium3DTileset::Tick(float DeltaTime) {
   for (Cesium3DTiles::Tile* pTile : result.tilesToRenderThisFrame) {
     if (pTile->getState() != Cesium3DTiles::Tile::LoadState::Done) {
       continue;
+		}
+     
+    // Consider Exclusion zone to drop this tile... Ideally, should be considered in Cesium3DTiles::ViewState to avoid loading the tile first...
+    if (ExclusionZones.Num() > 0) {
```

Comment:
We need a feature like this! And it's coming! But I don't think we want to ship something like this in the meantime, just like we didn't ship the previous rectangle-based clipping system I hacked in for Project Anywhere. The problems are:

* As you noted, it's inefficient to exclude tiles here because they've already been loaded. It needs to happen in cesium-native instead.
* This mechanism only works for tiles that use a "BoundingRegion" as their bounding volume. Lots of tilesets won't, especially photogrammetry. Also, as currently written, such tilesets will cause a crash if the user tries to define an exclusion zone on them (`pRegion` will be nullptr).
* It excludes way too much, especially for a tileset like Cesium OSM Buildings that uses additive refinement. Because the biggest buildings will be in tiles covering large areas, and those areas are more likely to intersect an exclusion zone, it's going to be fairly common for the largest buildings to be mysteriously missing just cause there's an exclusion zone elsewhere.

I know you're relying on this heavily for the HoloLens stuff, though, so maybe there's a middle ground. Could we include it in the shipped plugin but behind an `EXPERIMENTAL_EXCLUSION_ZONES` #define or something perhaps?

Reply:
I understand. 
I did it like that because I did not want to do edit Cesium3DTiles, but only in the plugin. What is important for now is to have the feature ON by default in the release build. So I'm ok if you put it in #defines and the implementation changes afterwards. 

But from an user point of view, can you keep the same UX ? I mean in the future implementation, you'll probably still be able to also define rectangular clipping zones. So if the property is there and just the way to process it changes for a better implementation later on, that's fine.

Reply:
> What is important for now is to have the feature ON by default in the release build.

Does that mean that if we wrap it in a #define, that #define has to be enabled by default?

> But from an user point of view, can you keep the same UX ?

Maybe! Let me think about that. It's always a bit risky to commit to an API before it has been thoroughly considered, which is exactly why we removed the old opacity mask based exclusion system before shipping 1.0.